token/cli/src/main.rs

```diff
@@ -415,19 +471,39 @@ fn command_create_multisig(
             minimum_signers,
         )?,
     ];
-    Ok(Some((
+
+    let tx_return = handle_tx(
+        &CliSignerInfo {
+            signers: bulk_signers,
+        },
+        config,
+        no_wait,
         minimum_balance_for_rent_exemption,
-        vec![instructions],
-    )))
+        instructions,
+    )?;
+    println!(
```

Comment:
What do you think about returning `Result<String>` from each of the the command fns, and handling the print altogether? (This is how solana-cli does it.)

I'll make a comment below about where.